### PR TITLE
Use only 2018 pyfrc versions

### PR DESCRIPTION
**Pull request by BobbyMi11er**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-pyfrc>=2018.2.3
+pyfrc>2019.0.0
```
Comment (ArchdukeTim) [inline requirements.txt]: your less than sign and my less than sign look different

Comment (ArchdukeTim) [reply requirements.txt]: But <2019.0.0 isn't entirely the correct answer. Think about why not

Comment (ErikBoesen) [reply requirements.txt]: Do you mean because it would include previous years? Does the `requirements.txt` format support giving a range of versions?

Comment (ArchdukeTim) [reply requirements.txt]: it does indeed. You can do pyfrc>=2018.0.0,<2019.0.0

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-pyfrc>=2018.2.3
+pyfrc>=2018.0.0,<2019.0.0
 robotpy-ctre>=2018.3.1
```
Comment (ErikBoesen) [inline requirements.txt]: Can you do the same thing for all the other robotpy packages with year versions?

**Pull request merged**